resources/readings/Languages.md

```diff
@@ -146,7 +146,7 @@ age = false; // !ok, false is not an int
 age = '10'; // !ok, '10' is a string, not an int
 ```
 
-In contrast, languages that use dynamic types, _values_ rather than variables have types. For example, in JavaScript:
+In contrast, in languages that use dynamic types, _values_ rather than variables have types. For example, in JavaScript:
```

Comment:
not sure if this is a fix - original wording seems more correct to me.

Reply:
Try reading the sentence without the "In contrast". Or replace "In contrast" with "On the other hand" or something like that

Anyway, it's not a big error. Feel free to close PR if you don't think it's a necessary change :)